source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
 
-  /gpfs/f5/epic
-  /gpfs/f5/gfdl_sd
+The following sections describe in detail how to create, submit, and manage
+jobs for execution on Frontier. Frontier uses SchedMD's Slurm Workload Manager
+as the batch scheduling system.
 
 
-Allocations and Quotas
+Login vs Compute Nodes
 ======================
 
-CPU allocations on Gaea are defined by the allocation board, with
-allocations allotted among different groups and systems. Each of these
-currently has a portion of time allocated. Dual running is done within
-the standard allocations under a QOS (Quality of Service) tag of
-"dual." Windfall is a catch-all quality of service account for users
-who have exhausted their groups monthly CPU allocation, or who wish to
-run without charging to their groups CPU allocation and forfeit job
-priority factors.
-
-SLURM is a Resource Manager and Scheduler. For Gaea-specific SLURM
-information, see SLURM Tips. For a general introduction to SLURM, see
-SLURM.
-
-
-Modules
-=======
-
-The Environment Modules system is a tool to help users manage their
-Unix or Linux shell environment, by allowing groups of related
-environment-variable settings to be made or removed dynamically.
-Modules provides commands to dynamically load, remove and view
-software.
+Recall from the `System Overview <#system-overview>`_ that Gaea contains two
+node types: Login and Compute. When you connect to the system, you are placed
+on a *login* node. Login nodes are used for tasks such as code editing,
+compiling, etc. They are shared among all users of the system, so it is not
+appropriate to run tasks that are long or computationally intensive on login
+nodes. Users should also limit the number of simultaneous tasks on login nodes
+(e.g. concurrent tar commands, parallel make).
 
-LMOD
-----
+Compute nodes are the appropriate place for long-running,
+computationally-intensive tasks. When you start a batch job, your batch script
+(or interactive shell for batch-interactive jobs) runs on one of your allocated
+compute nodes.
 
-LMOD is the modules software management system used on C5 and the C5
-login nodes. Unlike the module system on C3/C4, LMOD employs a
-hierarchical system that, when used properly, considers dependencies
-and prerequisites for a given software package. For example, the
-cray-netcdf module depends on the cray-hdf5 module and cannot be seen
-by the standard module avail commands, nor can it be loaded until the
-cray-hdf5 module is loaded.
+.. warning::
 
-The LMOD hierarchical system will automatically deactivate or swap an
-upstream module dependency. Two examples are given below.
+  Compute-intensive, memory-intensive, or other disruptive processes running on
+  login nodes may be killed without warning.
+
+Slurm
+=====
+
+Gaea uses `SchedMD <https://www.schedmd.com/>`_\ 's Slurm Workload Manager for
+scheduling and managing jobs. A few items related to Slurm are below.  See our
+:ref:`slurm-scheduler` documentation for more information.  You may also want
```

Comment:
When rendered this reads as "Slurm documentation" twice. Maybe add "scheduler" as a literal after the slurm-scheduler directive?